### DISCUSSION TITLE
Remove the version to appear in the title when performing your own version

### DISCUSSION THREAD

**Discussion by MarceloAraujo08** [nroduit/Weasis#485] [Q&A]
I would like to know if there was a command like -Dchangelist=-mybuild-beta to remove SNAPSHOT, to remove the appearance of the version

**Comment by nroduit** [comment]
It is essential to have a version of the jar files to ensure compatibility between them.
In addition, the package cache mechanism would work inconsistently.

I don't understand the intention behind this.
